src/PowerShellEditorServices/Services/PowerShellContext/Session/PSReadLinePromptContext.cs

```diff
@@ -98,7 +80,10 @@ internal class PSReadLinePromptContext : IPromptContext
             {
                 pwsh.Runspace = runspace;
                 var psReadLineType = pwsh
-                    .AddScript(ReadLineInitScript, useLocalScope: true)
+                    .AddCommand("Microsoft.PowerShell.Core\\Import-Module")
+                        .AddParameter("Name", _psReadLineModulePath)
+                    .AddStatement()
+                    .AddScript("[Microsoft.PowerShell.PSConsoleReadLine]", true)
```

Comment:
So reading through this again, it's not strictly required to do this part in PowerShell.

We could do it from C# instead like:

```csharp
pwsh
    .AddCommand("Microsoft.PowerShell.Core\\Import-Module")
    .AddParameter("Name", _psReadLineModulePath)
    .Invoke();

Type psReadLineType = Type.GetType("Microsoft.PowerShell.PSConsoleReadline");
```

Reply:
Heh, I think I like that more, yeah. But are we just yak shaving?

Reply:
Yeah this is a bit of bike shedding here, but there are some advantages to the latter form. The main one being performance is better (this code being on the startup path).

Reply:
Not gonna lie, I dig the GitHub integration in VS Code.

Reply:
Ok so doing that seems to cause the console to not have PSReadLine initialized. Like, it's imported, but not being used. Not sure the right term for that.

Reply:
Strange:

```
2021-07-01 10:54:54.222 -07:00 [INF] Attempting to load PSReadLine
2021-07-01 10:54:54.378 -07:00 [VRB] Loaded into load context "Default" System.Runtime.Loader.DefaultAssemblyLoadContext #0: Microsoft.PowerShell.PSReadLine2, Version=2.1.0.0, Culture=neutral, PublicKeyToken=null
2021-07-01 10:54:54.380 -07:00 [VRB] Loaded into load context "Default" System.Runtime.Loader.DefaultAssemblyLoadContext #0: mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
2021-07-01 10:54:54.466 -07:00 [WRN] PSReadLine unable to be loaded: <Unknown reason>
```

Reply:
But `Get-Module` lists it.

Reply:
Ok so we got that working, but adding a test for this is proving soooo difficult.